backend/src/controllers/social.controller.ts

```diff
@@ -16,6 +16,19 @@ export const getAll = async (_req: Request, res: Response) => {
   }
 };
 
+export const getAllActive = async (_req: Request, res: Response) => {
+    try {
+    const socials = await Social.find({active: true});
+    if (socials.length === 0) {
+      return res.status(StatusCode.NOT_FOUND).json({ message: 'Deactive Socials not found' });
```

Comment:
- [ ] Please change Incorrect error message 
- [ ] Please add this functionality to unit tests

Reply:
OK